COSC481W-2024Fall/MLite
# Nicholas fiori dataset viewer to preview first n rows of dataset

Features

- Added a dataset viewer.
- Added ability for dataset viewer to show first 20 and last 10 if CSV file contains more than 30 rows.

Testing

- Go to datasets.
- Click on Show This dataset for a previously uploaded dataset.
- Look towards the bottom to see the data preview.
- If the file contains more than 30 rows, it will show a message stating it only shows the first 20 and last 10.
